pages/common/golangci-lint.md

```diff
@@ -0,0 +1,16 @@
+# golangci-lint
+
+> Smart, fast linters runner for Go.
+> More information: <https://golangci-lint.run/usage/quick-start/>.
+
+- Run linters in current folder:
+
+`golangci-lint run`
+
+- List enabled/disabled linters (note: disabled linters are shown last, do not mistake them for enabled ones):
+
+`golangci-lint linters`
+
+- [E]nable a specific linter for this run:
+
+`golangci-lint run --enable linterToEnable`
```

Comment:
```suggestion
`golangci-lint run --enable {{linter_to_enable}}`
```
Should we prefer {{linter}} or {{linter_to_enable}}? Since it has enable before the placeholder, I think "linter" is fine

Reply:
I think {{linter}} is fine